xarray/tests/test_plot.py

```diff
@@ -2042,7 +2042,7 @@ def test_seaborn_palette_as_cmap(self) -> None:
     def test_convenient_facetgrid(self) -> None:
         a = easy_array((10, 15, 4))
         d = DataArray(a, dims=["y", "x", "z"])
-        g = self.plotfunc(d, x="x", y="y", col="z", col_wrap=2)
+        g = self.plotfunc(d, x="x", y="y", col="z", col_wrap=2)  # type: ignore[arg-type] # https://github.com/python/mypy/issues/15015
```

Comment:
This errors with 
```
xarray/tests/test_plot.py:2045: error: Argument "col" has incompatible type "str"; expected "None"  [arg-type]
```

self.plotfunc is a staticmethod and it seems that mypy only chooses one of the overloads for that plotfunc.
Might be related to https://github.com/python/mypy/issues/15015.

Reply:
Well, nothing we can do about it.

Reply:
Refactoring the tests without the staticmethod trick was an idea I had, but I'm not in the mood for such a large rewrite.

Reply:
Haha yes, I also thought about this several times already. Just too large of a refactor with almost no gains...